adal/build.gradle

```diff
@@ -201,4 +214,8 @@ task checkstyle(type: Checkstyle) {
     classpath = files()
 }
 
-assembleDebug.dependsOn 'checkstyle', 'pmd', 'findbugs', 'lint', 'createPom'
\ No newline at end of file
+tasks.whenTaskAdded { task ->
+    if (task.name == 'assembleDebug') {
+        task.dependsOn 'checkstyle', 'pmd', 'findbugs', 'lint', 'createPom'
+    }
```

Comment:
looks like we already were.

---
In reply to: [90352316](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90352316) [](ancestors = 90352316)